sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/DeployUpdate/DeployUpdate.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Constant.cs" Link="Constant.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.5.0" />
```

Comment:
We'll need to remove the version number - it gets pulled in from [package.data.props](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L93)

Reply:
So just: `<PackageReference Include="Azure.Identity" />` ?

Reply:
Yes, I added this as a suggestion below that you can commit from the PR.